README.md

```diff
@@ -34,13 +34,32 @@ In this exercise, we'll walk through executing a handful of common and handy SQL
 
 ## Code Along: SQL Queries
 
-### Preparing our Database
+### Creating our Database
 
-Once again we'll be using the `pets_database.db` from the previous exercises. Navigate to the directory that holds that database and connect to it with `sqlite3 pets_database.db`. 
+In this code along, we'll be creating a `cats` table in a `pets_database.db`. So, let's navigate to our terminal and get started.
 
-Let's add a few more cats to our `cats` table to make this more interesting: 
+First let's create our `pets_database` by running the following command.
+
+```bash
+sqlite3 pets_database.db
```

Comment:
Put in a quick "hey let's type LS and see what just happened, oh a new file appeared! This is the binary representation of the database. You can think of this like a `.jpg` file. It won't open up in a text editor, but it does open up in the image viewer app! just like this file